config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -113,7 +113,7 @@ presubmits:
             - ./test_e2e.sh
           env:
             - name: KIND_K8S_VERSION
-              value: "v1.25.3"
+              value: "v1.26.0"
```

Comment:
1.26.0 (https://github.com/kubernetes/kubernetes/issues/119840) had a major bug in client-go, since previously it wasn't affecting us, it shouldn't bother us. But to be on the safe side, can we use 1.26.6?